src/app/patient-search/patient.styles.css

```diff
@@ -0,0 +1,11 @@
+.wrapper {
+  width: 40%;
+  background-color: rgb(164, 164, 163);
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
can be removed, not needed background-color, width, margin-left and margin-right